tg_bot/modules/locks.py

```diff
@@ -0,0 +1,433 @@
+                        html.escape(chat.title),
+                        mention_html(user.id, user.first_name),
+                        args[0],
+                    )
+                )
+            else:
+                message.reply_text(
+                    "Cosa stai cercando ti sbloccare...? Usa /locktypes per la lista degli elementi bloccabili."
+                )
+
+        else:
+            bot.sendMessage(chat.id, "Cosa stai cercando ti sbloccare...?")
+
+    return ""
+
+
+@run_async
+@user_not_admin
+def del_lockables(bot: Bot, update: Update):
+    chat = update.effective_chat  # type: Optional[Chat]
+    message = update.effective_message  # type: Optional[Message]
+
+    for lockable, filter in LOCK_TYPES.items():
+        if (
+            filter(message)
+            and sql.is_locked(chat.id, lockable)
+            and can_delete(chat, bot.id)
+        ):
+            if lockable == "bots":
+                new_members = update.effective_message.new_chat_members
+                for new_mem in new_members:
+                    if new_mem.is_bot:
+                        if not is_bot_admin(chat, bot.id):
+                            message.reply_text(
+                                "Vedo un bot, mi è stato detto di non permettergli di unirsi... "
+                                "ma non sono un admin!"
+                            )
+                            return
+
+                        chat.kick_member(new_mem.id)
+                        message.reply_text(
+                            "Solo gli admin sono autorizzati ad aggiungere bot a questo gruppo! Fuori di qui."
+                        )
+            else:
+                try:
+                    message.delete()
+                except BadRequest as excp:
+                    if excp.message == "Message to delete not found":
+                        pass
+                    else:
+                        LOGGER.exception("ERROR in lockables")
+
+            break
+
+
+@run_async
+@user_not_admin
+def rest_handler(bot: Bot, update: Update):
+    msg = update.effective_message  # type: Optional[Message]
+    chat = update.effective_chat  # type: Optional[Chat]
+    for restriction, filter in RESTRICTION_TYPES.items():
+        if (
+            filter(msg)
+            and sql.is_restr_locked(chat.id, restriction)
+            and can_delete(chat, bot.id)
+        ):
+            try:
+                msg.delete()
+            except BadRequest as excp:
+                if excp.message == "Message to delete not found":
+                    pass
+                else:
+                    LOGGER.exception("ERROR in restrictions")
+            break
+
+
+def build_lock_message(chat_id):
+    locks = sql.get_locks(chat_id)
+    restr = sql.get_restr(chat_id)
+    if not (locks or restr):
+        res = "Attualmente non ci sono elementi bloccati in questo gruppo."
+    else:
+        res = "In questo gruppo questi sono gli elementi bloccati:"
```

Comment:
errore di traduzione sostituire con:
_res = "Gli elementi bloccati in questo gruppo sono i seguenti:"_